idd/Energy+.idd.in

```diff
@@ -50717,6 +50717,7 @@ Coil:Cooling:DX,
        \type alpha
        \reference CoilCoolingDX
        \reference AFNCoilNames
+       \reference DesuperHeatingCoilSources
```

Comment:
The IDD changes are the only thing I am unsure of since I can't get Git to open up the text above and below these locations to get a reference of where I am. When I click the arrows at the top header of this text block, nothing happens.